### ISSUE TITLE
Remove the included advertisement-blocker.

### ISSUE THREAD

**Issue by BEEDELLROKEJULIANLOCKHART** [kiwibrowser/src.next#693]
**Is your feature request related to a problem? Please describe.**
I do not want unnecessary functionality included that is provided better by an extension.

**Describe the solution you'd like**
Remove 

**Describe alternatives you've considered**
Replace and preinstall uBlock Origin instead of the current custom solution, or allow the current implementation to be uninstalled.

**Additional context**
I utilize uBlock Origin, so I possess no need for another solution.

**Comment by gnadelwartz** [comment]
You can switch internal AdBlock off

**Comment by HT-7** [comment]
Indeed, you can simply deactivate it. It's not like it prevents you from getting work done.

**Comment by BEEDELLROKEJULIANLOCKHART** [comment]
@gnadelwartz, how?
@HT-7, that does not contradict what I propose. I believe that it remains utterly unnecessary.

**Comment by gnadelwartz** [comment]
Settings - Website settings - advertisment - allow

**Comment by BEEDELLROKEJULIANLOCKHART** [comment]
@gnadelwartz, unfortunately that neither hides nor uninstalls the adblocker, but I am thankful, because it indeed disables it.
